pkg/csi/service/common/commonco/k8sorchestrator/k8sorchestrator.go

```diff
@@ -31,57 +34,125 @@ import (
 var (
 	k8sOrchestratorInstance         *K8sOrchestrator
 	onceFork8sOrchestratorInstance  sync.Once
-	featureStatesConfigMapName      string
-	featureStatesConfigMapNamespace string
+	supervisorFSSConfigMapName      string
+	supervisorFSSConfigMapNamespace string
+	internalFSSConfigMapName        string
+	internalFSSConfigMapNamespace   string
+	clusterFlavor                   cnstypes.CnsClusterFlavor
```

Comment:
All these can be moved to `K8sOrchestrator`

Reply:
`K8sOrchestrator` is an exported struct. Why do we need these in that?

Reply:
These are private variables and it should not be a problem moving it to `K8sOrchestrator`. Now the question is whether these fields belongs to `K8sOrchestrator` or not, IMO they do and they need not be defined as global variable.

Reply:
+1
Also, how about grouping `ConfigMapName`, `ConfigMapNamespace` and `FeatureStates` together into their own structure and having two variables of this type - one for internal fss and one for supervisor fss

Reply:
Refactored the code according to all the suggestions in this thread.